flink-runtime/src/test/java/org/apache/flink/runtime/executiongraph/restart/RestartStrategyFactoryTest.java

```diff
@@ -61,7 +61,7 @@ public void createRestartStrategyFactory_fixedDelayRestartStrategyConfigured_ret
 		final Configuration configuration = new Configuration();
 		configuration.setString(RestartStrategyOptions.RESTART_STRATEGY, "fixed-delay");
 		configuration.setInteger(RestartStrategyOptions.RESTART_STRATEGY_FIXED_DELAY_ATTEMPTS, attempts);
-		configuration.setString(RestartStrategyOptions.RESTART_STRATEGY_FIXED_DELAY_DELAY, delayBetweenRestartAttempts.getSeconds() + "s");
+		configuration.setString(RestartStrategyOptions.RESTART_STRATEGY_FIXED_DELAY_DELAY.key(), delayBetweenRestartAttempts.getSeconds() + "s");
```

Comment:
Wondering whether we should just use a `Duration` here.

Reply:
We can yes. It will no longer test the string parsing, but it does not have to. The parsing is covered in the `Configuration` tests.